vllm/entrypoints/openai/api_server.py

```diff
@@ -107,15 +108,14 @@ async def build_async_engine_client(args) -> AsyncIterator[AsyncEngineClient]:
     # Otherwise, use the multiprocessing AsyncLLMEngine.
     else:
         # Start RPCServer in separate process (holds the AsyncLLMEngine).
-        port = get_open_port(envs.VLLM_RPC_PORT)
         rpc_server_process = Process(target=run_rpc_server,
                                      args=(engine_args,
                                            UsageContext.OPENAI_API_SERVER,
-                                           port))
+                                           VLLM_RPC_PATH))
```

Comment:
We should probably add some kind of unique/random suffix here in case more than one instance of vLLM is run at a time.